packages/perf/lib/index.js

```diff
@@ -47,12 +56,35 @@ class FirebasePerfModule extends FirebaseModule {
   constructor(...args) {
     super(...args);
     this._isPerformanceCollectionEnabled = this.native.isPerformanceCollectionEnabled;
+    this._instrumentationEnabled = true;
   }
 
   get isPerformanceCollectionEnabled() {
     return this._isPerformanceCollectionEnabled;
   }
 
+  get instrumentationEnabled() {
+    return this._instrumentationEnabled;
```

Comment:
on iOS should return the value of this property https://firebase.google.com/docs/reference/ios/firebaseperformance/api/reference/Classes/FIRPerformance#instrumentationenabled

on android I'm not sure if there is any way to tell, should just return true I guess, with an implementation note saying the same in docs?

Reply:
Something about the local / javascript-level constants bothers me - in gray-box situations (where people have native code that might perturb the state) this could be out of sync with the actual native state of data collection enabled (both platforms) and instrumentation enabled (ios at least).

I think it is most correct to get rid of the javascript state holders and just always get the native state - and it would simplify things really. If Platform == android just always return true for instrumentation enabled but otherwise just always reach in and set them or reach in and get them from native